api/Metrics/ValueRecorder.php

```diff
@@ -7,7 +7,7 @@
 interface ValueRecorder
```

Comment:
I suggest we extend base Metrics interface ( https://github.com/open-telemetry/opentelemetry-php/blob/789819c9c146fca4989e618948183dcf7448ebfb/api/Metrics/Metric.php ) here in this PR

Reply:
Great catch, thank you.